ste/sourceInfoProvider-S3.go

```diff
@@ -82,6 +82,9 @@ func newS3SourceInfoProvider(jptm IJobPartTransferMgr) (ISourceInfoProvider, err
 			LogError: func(str string) { p.jptm.Log(pipeline.LogError, str) },
 			Panic:    func(err error) { panic(err) },
 		})
+	if jptm.MinimumLevelToLevel() == pipeline.LogDebug {
```

Comment:
Here as well. The appLogger which is part of jptm has enough context to filter logLevel, log level need not be inherited from jptm, you should just set pipeline.LogDebug as level in NewS3HTTPTraceLogger